ds_judgements_public_ui/templates/includes/recent_judgments.html

```diff
@@ -2,8 +2,15 @@
   <div class="recent-judgments__container">
     <h1 id="recently-published-judgments" class="recent-judgments__header">Recently published judgments</h1>
     {% include "includes/judgment_listing.html" with judgments=recent_judgments %}
-    <p class="judgment-listing__more-info">
-      <a href="{% url 'search' %}">See all recent judgments</a>
-    </p>
+    <section class="more-info-footer">
+      <div class="container py-2">
```

Comment:
Can we avoid using these helper classes (i.e. py-2 and container) and opt for styling in your component CSS. These styles should be applied to `more-info-footer`.

Reply:
@jacksonj04 is this a standard class introduced seen it also added in code by you previously so we can use this container py-2" as Tim also did ?

Reply:
If this is fine to keep in as I have to get to nect PR 3 then sort my IT issues for rest of this afternoon @jacksonj04 @jlhdxw